modules/Story/DiscourseComments.tsx

```diff
@@ -1,32 +1,40 @@
-import { FunctionComponent, useEffect } from 'react';
+import { useEffect } from 'react';
 
 type Props = {
-    topicId?: number
+    topicId?: number;
 };
 
-const DiscourseComments: FunctionComponent<Props> = ({ topicId }) => {
+const SCRIPT_ID = 'discourse-embed-script';
+
+export function DiscourseComments({ topicId }: Props) {
     useEffect(() => {
-        if (process.browser) {
-            // @ts-ignore
-            window.DiscourseEmbed = {
-                discourseUrl: 'https://discourse.prezly.io/',
-                topicId,
-            };
+        // @ts-ignore
+        window.DiscourseEmbed = {
```

Comment:
Can't you just declare this property on `window` instead of suppressing all TS errors?

Reply:
Oh yeah, I was going to do it, added the comment to allow the compilation to go through, but then totally forgot about it :)